docs/scripts/generate-sitemap-robotstxt.ts

```diff
@@ -35,7 +35,7 @@ async function generateSitemap() {
     <?xml version="1.0" encoding="UTF-8"?>
     <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:news="http://www.google.com/schemas/sitemap-news/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml" xmlns:mobile="http://www.google.com/schemas/sitemap-mobile/1.0" xmlns:image="http://www.google.com/schemas/sitemap-image/1.1" xmlns:video="http://www.google.com/schemas/sitemap-video/1.1">
           <url>
-            <loc>${process.env.SITE_URL}</loc>
+            <loc>${process.env.SITE_URL}/</loc>
```

Comment:
link without `/` should be redirected to the link with `/`, so add `/` to avoid the redirection